ktor-shared/ktor-serialization/ktor-serialization-gson/jvm/src/GsonConverter.kt

```diff
@@ -59,6 +84,33 @@ public class GsonConverter(private val gson: Gson = Gson()) : ContentConverter {
             throw JsonConvertException("Illegal json parameter found", deserializeFailure)
         }
     }
+
+    private val prettyPrinting by lazy {
+        val field = Gson::class.java.getDeclaredField("prettyPrinting")
```

Comment:
This can easily break if users use other versions of `Gson`. As an alternative, you can wrap it into `try/catch` or maybe just ignore pretty print for flows?

Reply:
> It's very counterintuitive that changing it can change response.

I agree with you, I chose to just ignore pretty print for the Flow case (this is mentioned in code comments keep track of it)